paddle/fluid/operators/fused/conv_fusion_op.cc

```diff
@@ -90,7 +111,8 @@ class Conv2DFusionOpInferShape : public framework::InferShapeBase {
       std::vector<framework::DDim> oshapes;
       oshapes.reserve(channels.size());
       for (size_t i = 0; i < channels.size(); ++i) {
-        oshapes.push_back({oshape[0], channels[i], oshape[2], oshape[3]});
+        oshapes.push_back(
+            {output_shape[0], channels[i], output_shape[2], output_shape[3]});
       }
       ctx->SetOutputsDim("Outputs", oshapes);
```

Comment:
The calculation logic of the above code is the same as [`ConvOp::InferShape`](https://github.com/PaddlePaddle/Paddle/blob/develop/paddle/fluid/operators/conv_op.cc#L33), and there are more checks on parameters in `ConvOp::InferShape` .
Better to use `ConvOp::InferShape`.

Reply:
when compute split kernels need output dims, cannot call conv::infershape directly.